comp/website/inform-teams.yaml

```diff
@@ -0,0 +1,24 @@
+summary: Inform teams of details of the competition
+
+priority: should
+
+component: Website
+
+milestone: $SRYYYY Competition
+
+area-owner: teams
+
+description: >-
+  We need to ensure teams know the important details of the competition
+
+  This must also include informing teams (probably by email) that photography and filming will take at the event,
+  to comply with our media consent policy.
+
+  Required actions:
+   - [ ] Publish the day schedule on the website
+   - [ ] Publish map of parking and directions to venue on website
+   - [ ] Publish maps of the competition layout (including team pit locations) on the website.
```

Comment:
I suggest we keep these as separate tasks or at least as sub-tasks. Since they're likely to get completed at different times, probably in different PRs which each want to "Fixes $issue", I think we're always going to want these to have their own sub-tasks. Having these as combined tasks also means that the dependencies are unclear -- for example the parking directions can be worked on before we have the day schedules & layout available.

Reply:
This is just the initial structure fro the task, the area owner responsible for doing it has the freedom (and tool support) to split them out into separate tasks if the choose. Having a single task here makes it easy to track what changes are needed.